Library/Formula/libass.rb

```diff
@@ -11,15 +11,23 @@ class Libass < Formula
     sha256 "7303bf3c119bc80d850f8e16ac1212e70224c83df3f8a3c0bf2adb0d378a6e80" => :mavericks
   end
 
+  option "without-coretext", "Disable CoreText backend in favor of the more traditional fontconfig"
+
   depends_on "pkg-config" => :build
   depends_on "yasm" => :build
 
   depends_on "freetype"
   depends_on "fribidi"
   depends_on "harfbuzz" => :recommended
+  depends_on "fontconfig" if build.without? "coretext"
```

Comment:
Let's keep the option as `with-fontconfig` and then a similar explanation. We're trying very hard to set fire to `if build.with?` style `depends_on` and I don't really want a new one added. Fontconfig can move to just being `:optional`.